awx/main/tasks.py

```diff
@@ -611,6 +617,7 @@ def awx_receptor_workunit_reaper():
     jobs_with_unreleased_receptor_units = UnifiedJob.objects.filter(work_unit_id__in=unit_ids).exclude(status__in=ACTIVE_STATES)
     for job in jobs_with_unreleased_receptor_units:
         logger.debug(f"{job.log_format} is not active, reaping receptor work unit {job.work_unit_id}")
+        receptor_ctl.simple_command(f"work cancel {job.work_unit_id}")
```

Comment:
From what I've observed, it is safe to call `work cancel` on a completed work item in receptor. We probably want to check the actual status of the job, however, and only cancel it if we need to.

Reply:
a work release command will first do a cancel, so I don't think this extra work cancel command is needed